pages/common/kubectl-config.md

```diff
@@ -0,0 +1,30 @@
+# kubectl config
+
+> Manage Kubernetes configuration for clients/clusters authentication to access clusters via `kubectl` or Kubernetes API.
+> By default the Kubernetes will get configuration in `${HOME}/.kube/config`.
+> A good tool that helps you better navigating in Kubernetes contexts (& namespace): <https://kubectx.dev>.
```

Comment:
```suggestion
```
IMO we should delete this line, because it refers to 3rd-party program that is not really an "official" Kubernetes tool.